doc/release-notes.md

```diff
@@ -42,21 +42,6 @@ From Bitcoin Core 22.0 onwards, macOS versions earlier than 10.14 are no longer
 Notable changes
```

Comment:
https://github.com/bitcoin/bitcoin/pull/25242 seems to be missing - not sure if on purpose?

Reply:
It's on line 58.

Reply:
Sorry I copied the wrong PR number, I meant #24256

Reply:
Included that now.